ktor-network/jvm/src/io/ktor/network/sockets/SocketOptions.kt

```diff
@@ -145,6 +145,11 @@ sealed class SocketOptions(
          */
         var keepAlive: Boolean? = null
 
+        /**
+         * Socket timeout. Default value is 0 that means infinite timeout.
+         */
+        var socketTimeout: Long = 0
```

Comment:
Consider using the same defaults as in the client

Reply:
Done.